integration_tests/helpers/apply.ts

```diff
@@ -151,6 +151,10 @@ export default class ApplyHelper {
     this.completeAccessAndHealthcareSection()
     this.completeFurtherConsiderationsSection()
     this.completeMoveOnSection()
+  }
+
+  completeApplication(isExceptionalCase?: boolean, isInComunity?: boolean) {
```

Comment:
I think it may be better for `completeApplication` to take another options argument like `isExceptionalCase` and `isInCommunity` that we can then pass into the `completeDocumentsSection` method called something like `verifyNoDocumentsMessage`. Then we can use it to decide whether to run the `shouldDisplayNoDocuments` method or the `shouldDisplayDocuments` method.  
Then we don't have to divide up the journey into two sections and it follows the established pattern.